client/src/components/TwitterMedia.js

```diff
@@ -0,0 +1,23 @@
+import React, {Component} from 'react';
+import { TwitterTimelineEmbed, TwitterShareButton, TwitterFollowButton, TwitterHashtagButton, TwitterMentionButton, TwitterTweetEmbed, TwitterMomentShare, TwitterDMButton, TwitterVideoEmbed, TwitterOnAirButton } from 'react-twitter-embed';
```

Comment:
You should only need to use `import {TwitterTweetEmbed} from 'react-twitter-embed'